static/app/views/replays/replayTable.tsx

```diff
@@ -0,0 +1,193 @@
+                avatarSize={32}
+                displayName={
+                  <Link
+                    to={`/organizations/${organization.slug}/replays/${generateEventSlug({
+                      project: replay.project,
+                      id: replay.id,
+                    })}/`}
+                  >
+                    {replay['user.display']}
+                  </Link>
+                }
+                user={{
+                  username: replay['user.display'],
+                  id: replay['user.display'],
+                  ip_address: replay['user.display'],
+                  name: replay['user.display'],
+                  email: replay['user.display'],
+                }}
+                // this is the subheading for the avatar, so displayEmail in this case is a misnomer
+                displayEmail={getUrlPathname(replay.url) ?? ''}
+              />
+              {isScreenLarge && (
+                <StyledPanelItem>
+                  <ProjectBadge
+                    project={
+                      projects.find(p => p.slug === replay.project) || {
+                        slug: replay.project,
+                      }
+                    }
+                    avatarSize={16}
+                  />
+                </StyledPanelItem>
+              )}
+              <StyledPanelItem>
+                <TimeSinceWrapper>
+                  {isScreenLarge && (
+                    <StyledIconCalendarWrapper color="gray500" size="sm" />
+                  )}
+                  <TimeSince date={replay.timestamp} />
+                </TimeSinceWrapper>
+              </StyledPanelItem>
+              {data.tableData ? (
+                <React.Fragment>
+                  <StyledPanelItem>
+                    <Duration
+                      seconds={
+                        Math.floor(
+                          dataEntries[replay.id]
+                            ? dataEntries[replay.id]['equation[0]']
+                            : 0
+                        ) || 1
+                      }
+                      exact
+                      abbreviation
+                    />
+                  </StyledPanelItem>
+                  <StyledPanelItem>
+                    {dataEntries[replay.id]
+                      ? dataEntries[replay.id]?.count_if_event_type_equals_error
+                      : 0}
+                  </StyledPanelItem>
+                </React.Fragment>
+              ) : (
+                <React.Fragment>
+                  <StyledPanelItem>
+                    <Placeholder height="24px" />
+                  </StyledPanelItem>
+                  <StyledPanelItem>
+                    <Placeholder height="24px" />
+                  </StyledPanelItem>
+                </React.Fragment>
+              )}
+            </Fragment>
+          );
+        });
+      }}
+    </DiscoverQuery>
+  );
+}
+
+const StyledPanelItem = styled('div')`
+  margin-top: ${space(0.75)};
+`;
```

Comment:
I know this is existing code but can we instead of giving it margin, make it flex with `align-items: center`? That'll be a more reliable way to vertically center the contents.